.github/workflows/md-link-check.yml

```diff
@@ -0,0 +1,34 @@
+name: Check Markdown links
+
+on:
+  pull_request:
+  push:
+    branches:
+      - main
+
+jobs:
+  markdown-link-check:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: checkout
+      uses: actions/checkout@v3
+      with:
+        fetch-depth: 0
+
+    - name: Check gh-pages
+      if: github.ref == 'refs/heads/gh-pages'
+      uses: gaurav-nelson/github-action-markdown-link-check@v1
+      with:
+        use-quiet-mode: 'yes'
+        use-verbose-mode: 'yes'
+        config-file: 'mlc_config.json'
+        file-path: './README.md'
+        folder-path: '_docs/'
+
+    - name: Check main
+      uses: gaurav-nelson/github-action-markdown-link-check@v1
+      with:
+        use-quiet-mode: 'yes'
+        use-verbose-mode: 'yes'
+        file-path: './README.md'
```

Comment:
So I'm clear. This checks all markdowns, including README.md? Why do you need to specify if `folder-path` defaults to all markdown files?

Reply:
Without any path it would check all the md files. However, b/c this repository has some stuff that I don't touch right now I'm making sure it only check the new docs under `_docs` in the gh-pages branch, and the README.md on both master and gh-pages.

In the future we can check more but I don't really know what is in the master, and the many other, branches.